discovery-infra/tests/config/global_configs.py

```diff
@@ -16,7 +16,7 @@ def get_default(key, default=None) -> Any:
 
 
 @dataclass
-class NodesConfig(BaseNodesConfig):
+class TerraformConfig(BaseTerraformConfig):
```

Comment:
Why? I think `Nodes` is better, no?

Reply:
There are arguments that only used by terraform, NodesConfig is a bit misleading name.